src/screen.cpp

```diff
@@ -300,10 +309,12 @@ void Screen::setVisible(bool visible) {
     if (mVisible != visible) {
         mVisible = visible;
 
-        if (visible)
-            glfwShowWindow(mGLFWWindow);
-        else
-            glfwHideWindow(mGLFWWindow);
+        if (mOwningGLFWContext) {
```

Comment:
Well, ``screen`` is a wrapper around the GLFW window, so ``setVisible()`` should be forwarded to GLFW. Whether the Screen also owns the GLFW window and is responsible for construction/deconstruction should not change the behavior of this function. Hiding the individual windows can be accomplished with their respective setVisible() functions.